astrobase/periodbase/kbls.py

```diff
@@ -1265,11 +1269,11 @@ def bls_stats_singleperiod(times, mags, errs, period,
 
         if magsarefluxes:
             blsmodel[transitindices] = (
-                blsmodel[transitindices] + thistransdepth
+                blsmodel[transitindices] - thistransdepth
```

Comment:
thanks for this correction. I need to check if the eebls.f returns +ve transit depths for fluxes though. Maybe you did this already.

Reply:
Yup, this is what led to the commit. I ran it with magsarefluxes==True, and got the wrong sign. (So I presumed it must have been the inverse for the magsarefluxes==False case).

![231663901_bls_fit](https://user-images.githubusercontent.com/13185735/45301334-bfe51180-b4de-11e8-8c36-0b5bc2f1c248.png)

Reply:
Ah good. So it's minuses for both cases then. I'll merge this in a bit.